main.js

```diff
@@ -68,23 +68,31 @@ app.on('ready', function () {
 
   if (process.platform === 'darwin') {
     // Create our menu entries so that we can use MAC shortcuts
-    Menu.setApplicationMenu(Menu.buildFromTemplate([
-      {
-        label: 'Edit',
-        submenu: [
-          { role: 'undo' },
-          { role: 'redo' },
-          { type: 'separator' },
-          { role: 'cut' },
-          { role: 'copy' },
-          { role: 'paste' },
-          { role: 'pasteandmatchstyle' },
-          { role: 'delete' },
-          { role: 'selectall' }
-        ]
-      }
-    ]));
-  }
+
+    var template = [{
+      label: app.getName(),
+      submenu: [
+        { label: "About " + app.getName(), selector: "orderFrontStandardAboutPanel:" },
```

Comment:
This is still incomplete as it doesn't include the Hide menu items. Also the Services menu should be included any time you have selectable text.

Is there a reason why you're not using `role` to define these menu items, e.g.:
```
      {role: 'about'},
      {type: 'separator'},
      {role: 'services', submenu: []},
      {type: 'separator'},
      {role: 'hide'},
      {role: 'hideothers'},
      {role: 'unhide'},
      {type: 'separator'},
      {role: 'quit'}
```